cmd/nodecmd/create_aws.go

```diff
@@ -95,16 +96,24 @@ func getAWSMonitoringEC2Svc(awsProfile, monitoringRegion string) (map[string]*aw
 	return ec2SvcMap, nil
 }
 
-func getAWSCloudConfig(awsProfile string) (map[string]*awsAPI.AwsCloud, map[string]string, map[string]int, error) {
+func getAWSCloudConfig(awsProfile string, singleNode bool, clusterSgRegions []string) (map[string]*awsAPI.AwsCloud, map[string]string, map[string]int, error) {
 	finalRegions := map[string]int{}
 	switch {
 	case len(numNodes) != len(utils.Unique(cmdLineRegion)):
 		return nil, nil, nil, fmt.Errorf("number of nodes and regions should be the same")
 	case len(cmdLineRegion) == 0 && len(numNodes) == 0:
 		var err error
-		finalRegions, err = getRegionsNodeNum(constants.AWSCloudService)
-		if err != nil {
-			return nil, nil, nil, err
+		if singleNode {
```

Comment:
we should not do this as it complicates logic on the func.
let's make getRegionsNodeNum work and keep getAWSCloudConfig unchanged